src/models/outbound/pispOTPValidate.model.ts

```diff
@@ -90,37 +91,35 @@ export class PISPOTPValidateModel
 
     this.logger.push({ channel }).info('onValidateOTP - subscribe to channel')
 
-    // eslint-disable-next-line no-async-promise-executor
-    return new Promise(async (resolve, reject) => {
-      let subId = 0
+    return deferredJob(this.pubSub, channel)
+    .init(async (channel) => {
+      const res = await this.thirdpartyRequests.patchConsentRequests(
+        consentRequestsRequestId,
+        { authToken: authToken }  as tpAPI.Schemas.ConsentRequestsIDPatchRequest,
```

Comment:
```suggestion
        { authToken }
```
the type casting to as `tpAPI.Schemas.ConsentRequestsIDPatchRequest` 
shouldn't be necessary here.